pkg/stores/mod/db.go

```diff
@@ -0,0 +1,76 @@
+package mod
+
+import (
+	ms "github.com/domino14/liwords/rpc/api/proto/mod_service"
+
+	"github.com/jinzhu/gorm"
```

Comment:
i know we have this all over the code but `"gorm.io/gorm"` is the new one. I don't think it's worth changing this at the moment since we have this everywhere.